src/orchard/meta.clj

```diff
@@ -104,13 +104,17 @@
 ;; They just map to a special symbol.
 (def special-sub-symbs '{& fn*, catch try, finally try})
 
-(defn repl-special-meta
+(defn repl-special-meta*
```

Comment:
Why `repl-special-meta*` instead of simply `special-meta*`?

Reply:
Well they are two different things in theory but let me double check if I am doing the right thing